client/web/antrea-ui/src/routes/summary.tsx

```diff
@@ -112,11 +248,111 @@ function ComponentSummary<T>(props: {title: string, data: T[], propertyNames: Pr
                         }
                     </tbody>
                 </table>
+                {isAgentTable && totalPages > 1 && (
+                    <div cds-layout="horizontal gap:sm align:center p-t:md">
+                        <CdsButton size="sm" action="outline" disabled={currentPage === 1} onClick={() => handlePageChange(currentPage - 1)}>Previous</CdsButton>
+                        <span cds-text="secondary">Page {currentPage} of {totalPages} ({filteredData.length} items)</span>
+                        <CdsButton size="sm" action="outline" disabled={currentPage === totalPages} onClick={() => handlePageChange(currentPage + 1)}>Next</CdsButton>
+                    </div>
+                )}
             </div>
         </CdsCard>
     );
 }
 
+function generateFakeAgents(n: number): AgentInfo[] {
```

Comment:
you should add unit tests in `summary.test.tsx` for the new sorting feature. The tests can use the synthetic data that you have here.

as a matter of fact, I think the test file already has some helper functions potentially to help with synthetic data generation?